reactor-test/src/main/java/reactor/test/util/RaceTestUtils.java

```diff
@@ -99,77 +101,79 @@
 		}
 	}
 
+	public static void race(final Runnable r1, final Runnable r2) {
```

Comment:
reintroduce the javadoc, mark as deprecated and move next to also deprecated `race(Runnable, Runnable, Scheduler)`?

Reply:
👍